File: 2025_09/leetcode2025-09-07.py
from typing import List

class Solution:
    def sumZero(self, n: int) -> List[int]:
        return list(range(-(n//2), 0, 1)) + [0]*(n%2) + list(range(1, n//2+1))

if __name__ == "__main__":
    s = Solution()
    n = 5
    print(s.sumZero(n))

File: leetcode2025/2025_09/leetcode2025-09-07.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> res;
        for (int i=1; i<=n/2; i++){
            res.push_back(i);
            res.push_back(-i);
        }
        if (n&1){
            res.push_back(0);
        }
        return res;
    }
};

int main(){
    Solution s;
    int n = 5;
    vector<int> res = s.sumZero(n);
    for (int i : res){
        cout << i << " ";
    }
    cout << endl;
}
